docs/accordproject-concepts.md

```diff
@@ -93,13 +93,32 @@ _More information about the Accord Project markup can be found in the [CiceroMar
 
 ![Template Model](assets/020/template_model.png)
 
-Compared to a standard document template (e.g., in Word or pdf), Accord Project templates associate a _model_ to the natural language text. The model lets you categorize variables (is it a number, a monetary amount, a date, a reference to a business or organization, etc.). This model is crucial in that it allows the computer to make sense of the information contained in your template.
+Compared to a standard document template (e.g., in Word or pdf), Accord Project templates associate a _model_ to the natural language text. The model acts as a bridge between the text and logic; it gives the users an overview of the components, as well as the traits of different components.
 
-The following screenshot shows the model for the acceptance of delivery clause above.
+The model lets you categorize variables (is it a number, a monetary amount, a date, a reference to a business or organization, etc.). This model is crucial, because it allows the computer to make sense of the information contained in your template.
 
-![Example Model](/img/model-vscode.png)
+The following shows the model for the **Acceptance of Delivery** clause above.
 
-Thanks to that model, the computer now knows that the `shipper` variable (`"Party A"` in the example above) contains a reference to an `Organization`, that variable `receiver` (`"Party B"` in the example above) also contains a reference to an Organization, that variable `deliverable` (`"Widgets"` in the example above) can contain any text description, etc.
+```ergo
+/* The template model */
+asset AcceptanceOfDeliveryClause extends AccordClause {
+  /** the shipper of the goods*/
+  --> Organization shipper
+  /** the receiver of the goods */
+  --> Organization receiver
+
+  /** what are we delivering? */
```

Comment:
> what is being delivered?

Removing "we"

Reply:
@irmerk @4katie As much as possible I would try to keep those examples be the exactly what's in the corresponding template. If we make changes here, then please make the same changes to the template in the template library as well.

Reply:
Ah didn't think to check that. Then skip this comment. I still think "we" should be avoided in the rest of the docs if possible.